### PR TITLE
chore: add renovate config

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**

### CODE REVIEW

File: test/open.test.js [hunk 1]
```diff
@@ -5,28 +5,6 @@ const { test } = require('node:test')
...
-    readyEmitted = true
-  })
-
-  setImmediate(() => {
-    assert.strictEqual(openEmitted, true)
-    assert.strictEqual(readyEmitted, true)
-    done()
-  })
-})
-
```
Comment (mcollina) [inline test/open.test.js:29]: why?

Comment (ivan-tymoshenko) [reply test/open.test.js:29]: The test is failing in the CI. It subscribes on evenets when the file is already opened. Do you know the correct way to test `open` and `ready` events?

Comment (mcollina) [reply test/open.test.js:29]: use https://github.com/mcollina/tspl

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**